freeipa/src/main/java/com/sequenceiq/freeipa/repository/StackRepository.java

```diff
@@ -76,6 +77,9 @@
     @Query("SELECT s FROM Stack s WHERE s.stackStatus.status IN :stackStatuses AND s.terminated = -1 ")
     List<Stack> findAllWithStatuses(@Param("stackStatuses") Collection<Status> stackStatuses);
 
+    @Query("SELECT s FROM Stack s WHERE s.stackStatus.detailedStackStatus IN :detailedStackStatuses AND s.terminated = -1 ")
```

Comment:
while `StackStatus#status` has index `detailedStackStatus` doesn't. Could you add it if we are going to use it in this where clause?